mautic/mautic
# [UX] Add a tooltip to explain what is Enclosure, Escape and Delimiter on Contacts import

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This update adds tooltips for the terms ‘Enclosure’, ‘Escape’, and ‘Delimiter’ in the Contacts import section. Recognizing the technical nature of these concepts, the tooltips provide straightforward explanations, aiding users in troubleshooting import issues. 

This enhancement fosters a more informative and user-friendly environment, empowering users to manage their imports with greater confidence and efficiency.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Open dropdown
4. Click to import
5. Hover the mouse on the labels

![image](https://github.com/mautic/mautic/assets/116097999/b755afe0-a6d8-484d-a0eb-c13f2299aaa6)
